src/main/java/seedu/duke/command/AddEventCommand.java

```diff
@@ -47,6 +49,8 @@ public AddEventCommand(Event event) {
     @Override
     public String execute() {
         timetable.addEvent(event);
-        return COMMAND_SUCCESSFUL_MESSAGE + Formatter.LS + Formatter.LS + event.toString();
+        ArrayList<String> eventStringArray = Formatter.formatEvent(event);
+        eventStringArray.add(0, COMMAND_SUCCESSFUL_MESSAGE);
+        return Formatter.formatString(eventStringArray, true);
```

Comment:
Maybe can overload another formatEvent to take in Header as one of the parameter. That way, just have to call 1 format function.